src/components/Explore/utils.ts

```diff
@@ -416,8 +421,15 @@ export function getChartIdByMetric(metric: ExploreMetric) {
   return exploreMetricData[metric].chartId;
 }
 
-export function getMetricLabels() {
-  return EXPLORE_METRICS.map(getTitle);
+export function getMetricLabels(normalizeData: boolean): string[] {
```

Comment:
rather than passing in `normlizeData`, i think we want to pass in a `selectedLocations.length > 1 `variable (so adding something like `const multiLocation = selectedLocations.length > 1` in Explore.tsx and passing that into `getMetricLabels`)

normalizeData takes `currentMetric` into account, and so the "per 100k" label only renders in the dropdown menu when a normalized metric is selected as `currentMetric`, ie:

<img width="1011" alt="Screen Shot 2021-06-12 at 1 08 28 PM" src="https://user-images.githubusercontent.com/44076375/121784086-e77dd280-cb7f-11eb-8935-a974ee203ad0.png">

but we'd want the "per 100k" label to render whenever multiple locations are selected, regardless of what `currentMetric` is

Reply:
oooooh, I see, thanks for that explanation

Reply:
fixed.